sdk/communication/communication-sms/samples/javascript/usingAadAuth.js

```diff
@@ -0,0 +1,65 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT Licence.
+
+/*
+  ONLY AVAILABLE IN NODE.JS RUNTIME
+  If you are using the browser, you can use the InteractiveBrowserCredential provided via @azure/identity or any other feasible implementation of TokenCredential.
+
+  This sample demonstrates how to instantiate SmsClient using AAD token credentials.
+
+  Setup :
+    Please ensure that your Communication Services resource is in US East, US East 2, or West Europe
+    region. AAD Role Based Access Control is not supported in other regions yet.
+
+    Register a new application in AAD
+      - See https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app
+        to register a new application in the Azure Active Directory.
+      - Note down the CLIENT_ID and TENANT_ID from the above step.
+      - In the "Certificates & Secrets" tab, create a secret and note that down.
+      - In the Azure portal, go to your Communication Services resource and click on the Access control (IAM)
+        tab. Here, assign the "Owner" role to the registered application.
+      
+    - Environment setup for the sample
+      - From the overview page of your AAD Application, note down the `CLIENT ID` and `TENANT ID`. In the "Certificates & Secrets" tab, create a secret and note that down.
+      - Make sure you have AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET as environment variables to successfully execute the sample(Can leverage process.env).
```

Comment:
Good find, and nice copy paste! :-)

```suggestion
      - Make sure you have AZURE_TENANT_ID, AZURE_CLIENT_ID, AZURE_CLIENT_SECRET as environment variables to successfully execute the sample (can leverage process.env).
```